lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -47,6 +52,16 @@ def getPrettyName(name):
     # Make each word start with an uppercase.
     return prettyName.title()
 
+# Helper class to push/pop the Attribute Editor Template. This makes
+# sure that controls are aligned properly.
+class AEUITemplate:
+    def __enter__(self):
+        cmds.setUITemplate('attributeEditorTemplate', pst=True)
+        return self
+
+    def __exit__(self, mytype, value, tb):
+        cmds.setUITemplate(ppt=True)
```

Comment:
Simple helper that will push/pop the attribute editor template (which has proper sizes set for the various layouts/controls).